### PR TITLE
Tabs format

### PR DESCRIPTION
**Pull request by TyBalduf**
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

**Changes to gaussian.py:**
* Uses f-string formatting throughout
* Use global lists/dicts of standard options to expedite formation of userOptions
* Creates a separate tab for Alternate Basis Set/Theory options.

### CODE REVIEW

**Comment by ghutchis** [comment]
Sorry, I just saw this PR. IIRC f-strings came in Python 3.6. I think we were targeting 3.5 and later, but it's worth discussing (e.g., https://discuss.avogadro.cc/) since I'm generally using 3.7 or later.

Otherwise, this looks good to go.

**Comment by cclauss** [comment]
https://devguide.python.org/#status-of-python-branches v.s.
https://devguide.python.org/devcycle/#end-of-life-branches

File: gaussian.py [hunk 1]
```diff
@@ -99,43 +89,40 @@ def generateInputFile(opts):
...
     theory = opts['Theory']
-    if opts['Alternate Basis Set'] == True:
-        basis = opts['Alternate Basis Set Name']
-    else:
-        basis = opts['Basis']
+    if opts["Alternate Theory"]:
+        theory = opts["Alternate Theory"]
+    basis = opts['Basis']
+    if opts['Alternate Basis Set']:
+        basis = opts['Alternate Basis Set']
```
Comment (cclauss) [inline gaussian.py:96]:
```suggestion
    basis = opts['Alternate Basis Set'] or opts['Basis']
```

**Comment by ghutchis** [comment]
After the discussion of v3.6 and f-strings, I'm good to go. Please complete the DCO sign-off:
https://github.com/OpenChemistry/avogenerators/pull/9/checks?check_run_id=3473894896

Thanks!

**Pull request merged**